fabfile/alba.py

```diff
@@ -356,34 +509,50 @@ def is_local(osd):
             return True
         return False
 
-    def inner():
+    def inner(abm_cfg):
+        if _use_tls(tls):
+            abm_cfg = TLS['arakoon_config_file'] #TODO:run_tests_recovery ?
```

Comment:
This logic doesn't belong here ... overriding an argument given by the caller is a bit weird.
I suggest lifting this part to the top of claim_local_osds.